tools/cjxl_ng_main.cc

```diff
@@ -35,9 +46,9 @@ DEFINE_bool(encoder_version, false,
             "Print encoder library version number and exit.");
 
 DEFINE_bool(add_jpeg_frame, false,
-            // This supersedes --jpeg_transcode
-            "Use JxlEncoderAddJPEGFrame to add a JPEG frame, "
-            "rather than JxlEncoderAddImageFrame.");
+            "Use JxlEncoderAddJPEGFrame to add a JPEG frame "
+            "(i.e. transcoding JPEG), rather than using "
+            "JxlEncoderAddImageFrame.");
 
 DEFINE_bool(jpeg_store_metadata, false,
```

Comment:
Wouldn't it make more sense to let this default to `true` so recompressed jpegs can actually be reconstructed? That's the current behavior of `cjxl`.

Reply:
? Duplicate comment?

Reply:
Ah! We are talking about jpeg_store_metadata. Agree that this should be default-true.